lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1556,6 +1556,10 @@ void hlsl::SerializeDxilContainerForModule(DxilModule *pModule,
 
   // If metadata was stripped, re-serialize the module.
   CComPtr<AbstractMemoryStream> pInputProgramStream = pModuleBitcode;
+  if (Flags & SerializeDxilFlags::StripReflectionFromDxilPart) {
+    bModuleDirty = true;
+    pModule->StripReflection();
```

Comment:
This looks like it will affect the debug module too, is there a way we can avoid that?

Reply:
Updated in commit 2.